src/org/javaday/java8/misused/stream/ExternalCollectionForGrouping.java

```diff
@@ -19,11 +19,9 @@
 
 import static java.util.stream.Collectors.*;
 
-/**
- * @author Alimenkou Mikalai
- * @version 1.0
- */
+//TODO: Decide whether to show or not as it's a bit complex example as for me
```

Comment:
We can if we have time at the end, but still think if it can be simplified in some way...